backend/Spring Boot/README.md

```diff
@@ -0,0 +1,34 @@
+# Getting Started with Spring Boot Technology
+
+Get started learning Spring boot with templete  and basic funtionalities releted to Spring Boot provided in this folder.
+
+## 📌 Introduction
+
+Spring Boot is an open source Java-based framework used to create a Micro Service. Developing Services with spring Boot is too cool.
+
+Spring Boot makes it easy to create stand-alone, production-grade Spring based Applications that you can “just run”. Most Spring Boot applications need very little Spring configuration.
+
+
+## 📌 How to get deep dive with Spring Boot?
+
+To start working with Spring Boot, we will require to do basic installations.
+
+[Installing Java](https://www.java.com/en/download/help/download_options.html)<br>
+[Installing IntelliJ IDEA](https://www.jetbrains.com/help/idea/installation-guide.html)
+
+Checkout starter project and you will be ready to implement Spring Boot as your backend technology to create Services.<br>
+
+In this Repository, you will find the cool things about Spring Boot and Spring Boot Starter Projects.Basic REST Services will be made available to this folder .
+
+## 📌 Goal
+Provide quick start for projects with flavour of Spring Boot .<br>
+Provide a range of features that are common to large classes of projects
+
+
+## 📌 Prerequisite to use Spring Boot

```

Comment:
Some direct links here to understand Annotations, basics of Spring, etc. will be helpful.